### PR TITLE
Use correct path when installing go-bindata

### PR DESCRIPTION
**Pull request by li-ang**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind api-change
> /kind bug
/kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

Use correct path when installing go-bindata

When using `go get -u xxx/xxx/...`,  the path like` xxx/xxx/...` is ok. 

But if we use `go install xxx/xxx/...`, it could be made error.

```
➜  kubernetes git:(master) make generated_files
warning: ignoring symlink /root/goproject/kubernetes/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes
go: warning: "k8s.io/kubernetes/vendor/github.com/go-bindata/go-bindata/..." matched no packages
➜  kubernetes git:(master) go version
go version go1.13.8 linux/amd64
➜  kubernetes git:(master) echo $GOPATH
/root/goproject/kubernetes
➜  kubernetes git:(master) echo $PATH
/root/goproject/kubernetes/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/usr/local/java/bin:/usr/local/go/bin:/root/golang/bin
➜  kubernetes git:(master) pwd
/root/goproject/kubernetes/src/k8s.io/kubernetes
➜  kubernetes git:(master) go install k8s.io/kubernetes/vendor/github.com/go-bindata/go-bindata/...
go: warning: "k8s.io/kubernetes/vendor/github.com/go-bindata/go-bindata/..." matched no packages
➜  kubernetes git:(master)
```

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by li-ang** [comment]
@liggitt

File: hack/generate-bindata.sh [hunk 1]
```diff
@@ -33,7 +33,7 @@ export GOBIN="${KUBE_OUTPUT_BINPATH}"
 PATH="${GOBIN}:${PATH}"
 
 # Install tools we need, but only from vendor/...
-go install k8s.io/kubernetes/vendor/github.com/go-bindata/go-bindata/...
+go install k8s.io/kubernetes/vendor/github.com/go-bindata/go-bindata
```
Comment (liggitt) [inline hack/generate-bindata.sh]: it's actually `k8s.io/kubernetes/vendor/github.com/go-bindata/go-bindata/go-bindata`

Comment (li-ang) [reply hack/generate-bindata.sh]: Oh, THX!

**Pull request merged**